packages/flutter/lib/src/material/card.dart

```diff
@@ -120,6 +121,11 @@ class Card extends StatelessWidget {
   /// if that's null then [ThemeData.cardColor] is used.
   final Color color;
 
+  /// The color to paint the shadow below the card.
+  ///
+  /// Defaults to fully opaque black.
```

Comment:
If null then the ambient [CardTheme]'s shadowColor is used. If that's null too, then the default is fully opaque black.